src/main/content/_assets/js/javadoc-redirect.js

```diff
@@ -65,4 +76,19 @@ $(function () {
       '/package-frame.html&class=' +
       jd;
     }
+    else {
+      ifPackageNotExists(port,'liberty-jakartaee',version,jd);
+    }
+  }
 });
+
+function ifPackageNotExists(port,api,version,jd) {
+  window.top.location.href =
+  'https://' +
+  window.top.location.hostname +
+  port +
+  '/docs/latest/reference/javadoc/'+ api +
+  version +
+  '-javadoc.html?class=' +
+  jd;
+}
```

Comment:
Nit-pick: This file needs an end of line

Reply:
added